package.json

```diff
@@ -74,14 +74,14 @@
     "yosay": "^2.0.2"
   },
   "devDependencies": {
+    "c8": "^10.1.3",
     "coveralls": "^3.0.2",
     "mocha": "^11.0.1",
     "mockery": "^2.0.0",
     "nock": "^13.2.1",
-    "nyc": "^17.1.0",
-    "proxyquire": "^2.0.1",
     "registry-url": "^5.1.0",
     "sinon": "^19.0.2",
+    "testdouble": "^3.20.2",
```

Comment:
Also considered sinon and rewiremock as alternatives here but this was the most seamless.

If this were my project, I'd switch to jest and use their module mocking.

Reply:
We have adopted vitest in some packages.